backend/models/test.model.ts

```diff
@@ -98,6 +106,14 @@ export interface Test extends Document {
   questions: QuestionComponent[][];
   /** The intended grade the test was made for */
   grade: number;
+  /** the type of assessment */
```

Comment:
can you match the documentation in `backend/services/interfaces/testService.ts` with this file